miner/src/block_assembler.rs

```diff
@@ -286,6 +286,11 @@ impl<'a> BlockAssembler<'a> {
 			// memory pool should restrict miner fee to be positive
 			miner_reward += entry.miner_fee as u64;
 			let tx = IndexedTransaction::new(entry.hash.clone(), entry.transaction.clone());
+			if let Some(ref sapling) = tx.raw.sapling {
+				for out in &sapling.outputs {
+					sapling_tree.append(out.note_commitment.into()).expect("TODO");
```

Comment:
Should probably be plain panic anyway, so "TODO" can be replaced with sensible message